docker-compose.yml

```diff
@@ -22,6 +46,53 @@ services:
         extends:
             file: common.yml
             service: mender-base
+        labels:
+            - traefik.enable=true
+            - traefik.http.routers.gui.entrypoints=https
+            - traefik.http.routers.gui.middlewares=ensure-ui-path,signup-redirect,ui-stripprefix,sec-headers,json-error-responder1,json-error-responder2,json-error-responder3,json-error-responder4
+            - traefik.http.routers.gui.rule=PathPrefix(`/`)
+            - traefik.http.routers.gui.service=gui
+            - traefik.http.routers.gui.tls=true
+            - traefik.http.services.gui.loadbalancer.server.port=80
+            
+            - traefik.http.routers.error-responder.entrypoints=https,http
+            # - traefik.http.routers.error-responder.rule=HostRegexp(`{host:.+}`)
```

Comment:
if I remember correctly that's the error catch-all - based on the passing tests I think the error responses seem to work just fine, so I don't think we need that line to be in 🤷‍♂️